test/e2e/config/rabbitmqvhost/clustervhost.yaml

```diff
@@ -0,0 +1,28 @@
+# Copyright 2021 The Knative Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: rabbitmq.com/v1beta1
+kind: RabbitmqCluster
+metadata:
+  name: rabbitmqc
+  namespace: {{ .namespace }}
+spec:
+  rabbitmq:
+    additionalConfig: |
+      default_vhost = test-vhost
+      default_user = guest
+      default_pass = guest
```

Comment:
is it safe to let a password be in the rabbitmq cluster CO? oh, this is not our code or resource. yikes, ok.

Reply:
Dont know if oook XD but is an option that is usefull for this test purpose, ill try to improve this over the weekend like to try to refer the default user secret on the config, help is welcome :p